### PR TITLE
/containers/docker-ubuntu page

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Add docker page content

## QA

Make sure the page contains the same content as the copy doc /containers/docker-engine-ubuntu

## Issue / Card

Card: https://trello.com/c/pbT71tGX
Copy doc: https://docs.google.com/document/d/18rdWBRLM6jYWilpT4-NSs6yFEO4aqtO_pHidea59Kfw/edit#

### CODE REVIEW

**Comment by grahambancroft** [comment]
[Containers: Docker page](https://trello.com/c/pbT71tGX/2545-containers-docker-page)

File: templates/containers/docker-engine-ubuntu.html [hunk 1]
```diff
@@ -15,9 +15,83 @@
...
+    </div>
+
+    <div class="box six-col last-col equal-height--item">
+      <h3>Enhanced security</h3>
+      <p>Packaging Docker Engine as a snap delivers additional isolation and confinement benefits which significantly improve the security of your Docker app.</p>
+    </div>
+
+    <div class="box six-col equal-height--item">
+      <h3>Safer, simpler roll backs</h3>
+      <p>If needed, you can easily undo an update and reinstall an older Docker snap that worked..</p>
```
Comment (anthonydillon) [inline templates/containers/docker-engine-ubuntu.html]: Single full stop please.

**Comment by anthonydillon** [comment]
QA issues:
- The URL of the page in the doc is: http://www.ubuntu.com/containers/docker-ubuntu
- Missing "The no-nonsense way to accelerate your business with containers" section

**Comment by deadlight** [comment]
The missing section is intentional, I've marked the copydoc.

**Comment by anthonydillon** [comment]
Ok thanks @deadlight.

**Comment by grahambancroft** [comment]
@anthonydillon  I’ve made all the changes

**Comment by grahambancroft** [comment]
Hi @anthonydillon this is ready for review again if you get a chance

**Pull request merged**